README.md

```diff
@@ -185,3 +186,82 @@ make test_unit
 # Shannon E2E test only
 make test_e2e_shannon_relay
 ```
+
+## User Data
+
+By default, PATH does not associate user data with service requests (i.e. relays).
+
+You may opt to enable user data config to unlock the ability to associate a specific user with a particular service request.
+
+This is required for:
+
+- User-specified app settings (e.g. secret key, allowlists, etc.)
+- Metering and billing of service requests (e.g. charging users $2 per 1 million requests)
+- Rate limiting of service requests by throughput (e.g. 30 req / second) and/or capacity (e.g. 1M req / month)
+
+### Updated Endpoint
```

Comment:
I feel like whatever IDE you're using still doesn't have the "auto update table of contents" feature. The ToC should've automatically updated if you did.

Reply:
Sure, this is a great add-on. I've added to my IDE.